react-components/src/hooks/useSearchMappedEquipmentAssetMappings.tsx

```diff
@@ -45,7 +46,7 @@ export const useSearchMappedEquipmentAssetMappings = (
         return modelsAssets.map(({ assets }) => assets).flat();
       }
 
-      const searchedAssets = await sdk.assets.search({ search: { query }, limit });
+      const searchedAssets = await sdk.assets.search({ search: { query }, limit: 1000 });
```

Comment:
Was just wondering what would happen if search result is more than 1000 as we do not have pagination, should we consider it?

Reply:
There is no pagination support for the search endpoint, so I suppose we just cross our fingers in that case...